### PR TITLE
Allow php 8

### PR DESCRIPTION
**Pull request by holtkamp**
Allow installing this provider on PHP 8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
     ],
     "homepage": "https://github.com/swisnl/geocoder-php-nationaal-georegister-provider",
     "require": {
-        "php": "^7.0",
+        "php": "^7.4 || ^8.0",
```
Comment (JaZo) [inline composer.json]: I'd like to keep PHP 7.0 as minimum required version for now.

**Review by JaZo — CHANGES_REQUESTED** [review]
Nice work @holtkamp! However, I'd like to keep PHP 7.0 as minimum required version for now. Also don't forget to update CI config (.travis.yml and .scrutinizer.yml) to run tests on all supported PHP versions.

**Comment by holtkamp** [comment]
I am afraid 7.2 will be the minimum since PHPUnit requires / allows this and higher versions (required for PHP 8) since [`8.5.12`](https://github.com/sebastianbergmann/phpunit/blob/f40cf5be80a7d177494592ad1b2db3f64ea1dee3/composer.json#L24), I could give that a try...?

But 7.4 gives all the niceness of typing, etc, so that would be my suggestion. A lot of libraries use 7.4 as minimum now. 

The Geocoder itself aims at 7.3:

https://github.com/geocoder-php/Geocoder/blob/3249d994244d30f591a71b1911ca936bd35f71b4/composer.json#L20

Also note that security fixes [have not been supported anymore for 7.0 since january 2019](https://www.php.net/supported-versions.php)

**Comment by JaZo** [comment]
I see supporting PHP 7.0 will need some workarounds and I agree it's not worth it. I'm aware of all the niceness in PHP 7.4, but for this one class library I don't feel the urge to have those available so then I'd like to go for PHP 7.2 as minimum.

**Comment by holtkamp** [comment]
Tried it with `"php": "^7.2 || ^8.0"` and `"phpunit/phpunit": "^8.5"` but then [`geocoder-php/provider-integration-tests`](https://github.com/geocoder-php/provider-integration-tests) is unable to install since none of the tagged versions allows php unit `8.5`:

```
Problem 1
    - geocoder-php/provider-integration-tests 1.5.0 requires phpunit/phpunit ^9.5 -> satisfiable by phpunit/phpunit[9.5.0, 9.5.1, 9.5.x-dev] but these conflict with your requirements or minimum-stability.
    - geocoder-php/provider-integration-tests 1.4.1 requires phpunit/phpunit ^7.5 -> satisfiable by phpunit/phpunit[7.5.0, 7.5.1, 7.5.10, 7.5.11, 7.5.12, 7.5.13, 7.5.14, 7.5.15, 7.5.16, 7.5.17, 7.5.18, 7.5.19, 7.5.2, 7.5.20, 7.5.3, 7.5.4, 7.5.5, 7.5.6, 7.5.7, 7.5.8, 7.5.9] but these conflict with your requirements or minimum-stability.
```

The combination `"php": "^7.3 || ^8.0"` and `"phpunit/phpunit": "^9.0"` works and installs PHPUnit `9.5`, so I guess the minimum version of PHP is forced to be `7.3`...?

**Comment by JaZo** [comment]
The testsuite is compatible with PHPUnit 7-9, so we can use `"phpunit/phpunit": "^7.0|^8.0|^9.0"` to allow running tests on PHP 7.2.

**Review by JaZo — APPROVED** [review]

**Comment by JaZo** [comment]
Thanks for your effort @holtkamp, released in [1.3.0](https://github.com/swisnl/geocoder-php-nationaal-georegister-provider/releases/tag/1.3.0)!

**Pull request merged**